packages/protocol/src/types.ts

```diff
@@ -1,34 +1,26 @@
-import type { DataContext, ValidationMessage } from '.';
+import type { DataClassEditorDataContext } from '.';
 import type { DataClass } from './dataclass';
 import type {
+  DataActionArgs,
   DataClassCombineArgs,
-  DataClassEditorDataContext,
+  DataClassData,
   DataclassType,
   JavaType,
   TypeSearchRequest,
   ValidationResult
 } from './editor';
 
-export { type DataClassEditorDataContext as DataContext };
-export type Data = { context: DataContext; data: DataClass };
-export type EditorProps = { context: DataContext; directSave?: boolean };
-export type SaveArgs = Data & { directSave?: boolean };
-export interface DataClassActionArgs {
-  actionId: 'openForm' | 'openProcess';
-  payload: string;
-  context: DataContext;
-}
-
-export { type ValidationResult as ValidationMessage };
```

Comment:
no big deal; but removing these types gives us more confident that the interfaces is accurate and sync with the backend.